annotator-core/src/main/java/edu/ucr/cs/riple/core/Report.java

```diff
@@ -46,17 +46,18 @@ public class Report {
   public Fix root;
   /** Fix tree associated to this report instance. */
   public Set<Fix> tree;
-  /**
-   * Set of fixes that will be triggered in target module if fix tree is applied to the source code.
-   */
-  public ImmutableSet<Fix> triggeredFixes;
   /**
    * Set of errors that will be triggered in target module if fix tree is applied to the source
    * code.
    */
   public ImmutableSet<Error> triggeredErrors;
-  /** If true, all leaves of fix tree are not resolvable by any {@code @Nullable} annotation. */
-  public boolean finished;
+  /**
+   * Set of triggered fixes on target module that will be triggered if fix tree is applied due to
+   * errors in downstream dependencies.
+   */
+  public ImmutableSet<Fix> triggeredFixesOnDownstream;
+  /** If true, this report has passed one iteration */
+  public boolean opened;
```

Comment:
Why call this `opened`?

Reply:
I wanted to distinguish between a report object that has just been created and one that has been processed for at least one iteration. Renamed it to `hasBeenProcessedOnce`.  2bdd664ec26ba0d5536ac345547550be26698254.

I didn't want to create a separate enum for this as it only has two cases and can be implemented with a single boolean. But if you think enum is a better design here, please let me know and I will update it. Thank you.